#6474: Align progress and help icons on to-dos column

keysmashes opened the pull request:
As per #6474, progress and help icons on the to-dos column were previously misaligned due to inconsistent inline styles. This commit adds the same inline styles to both icons (namely, `zoom: 1.5; vertical-align: -webkit-baseline-middle;`).

I haven't run any tests (can't get Vagrant working), but it's a one-line copy-paste... what can go wrong?

keysmashes committed:
Align progress and help icons on to-dos column

As per #6474, progress and help icons on the to-dos column were
previously misaligned due to inline styles. This commit adds the same
inline styles to both icons.

Alys commented:
You'd be surprised at what can go wrong with changes that seem simple. :)

If you want help getting Vagrant running, you can post here or in the Aspiring Coders guild. Once it's running, show us a screenshot of the effect of this change to help the staff approve it.

crookedneighbor commented on website/views/shared/tasks/task_view/graph.jade:
```diff
@@ -1,6 +1,6 @@
 span.option-box.pull-right(ng-if='::main')
   a.option-action(ng-if='list.type=="todo"', ng-show='obj.history.todos', ng-click='toggleChart("todos")', tooltip=env.t('progress'), style='margin-right:5px;')
-    span.glyphicon.glyphicon-signal
+    span.glyphicon.glyphicon-signal(style={'zoom':1.5,'vertical-align':'-webkit-baseline-middle'})
```
Why are there curly braces in this inline style? Also, we avoid inline styles, would prefer a class.

keysmashes replied:
I basically copy-pasted from line 9; which file should new styles go in?

keysmashes commented:
Edit: have got it working, see next comment.

~~Vagrant now seems to work fine, but `npm start` doesn't. Output is [here](http://hastebin.com/raw/ezofatepoh), and /vagrant/npm-debug.log is [here](http://hastebin.com/raw/iforuqirot).~~

~~I've tried the method mentioned in Aspiring Coders of `sudo apt-get update && sudo apt-get upgrade && sudo apt-get install curl && sudo curl -sL https://deb.nodesource.com/setup_0.12 | sudo sh && sudo apt-get install nodejs`, followed by `sudo npm install -g` and `sudo npm install -g [missingpackage]` over and over, but to no avail.~~

keysmashes commented:
What it currently looks like:

![image](https://cloud.githubusercontent.com/assets/9433472/12272196/02351b7e-b957-11e5-87e8-f926c439dcca.png)

What it looks like with this PR:

![image](https://cloud.githubusercontent.com/assets/9433472/12272206/0d90874c-b957-11e5-8b10-d16c2a13d83e.png)

If someone can point me towards the right file to put styles in (I'm guessing `website/public/css/tasks.styl`, but confirming that would be great), I'm happy to move the inline styles to a class.

Commit:
Move inline styles on task column icons to a class

This uses existing classes instead of adding a new one, because if new
icons are added in the future this should catch them without any extra
classes having to be added to the new icons.

In addition, this will catch any new .option-box added to .task-column
in the future; this may be good or bad, but is easy to fix in any case.

crookedneighbor commented:
@Alys @lemoness How do you feel about the enlarged progress graph icon?

lemoness commented:
Looks much better!

Alys commented:
Is good!

crookedneighbor commented:
Merged in with https://github.com/HabitRPG/habitrpg/commit/a0b264f6860f59fec71b0c998e2ec1e07991bc11

This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

keysmashes commented:
@crookedneighbor: User ID is `0e1d15f7-f36b-4f15-a9e1-0c31b732dac9`. Thanks :)

crookedneighbor commented:
Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!
